src/index.bitclout.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
We should provide some guidance in the readme about how to modify this file for your own app since it's a little more manual than just updating the environment file.